src/app/ArrayValueCalculator.java

```diff
@@ -33,4 +33,13 @@ public int doCalc(String[][] array) {
 
         return sum;
     }
+
+    private boolean isEachRowOfSize(String[][] array, int size) {
+        for (String[] row : array) {
+            if (row.length != size) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
in case the array will be 2X4  it will still be true. This behavior is not expected (incorrect )